rating-service/src/main/java/com/vlad/kuzhyr/ratingservice/utility/constant/ExceptionMessageConstant.java

```diff
@@ -0,0 +1,17 @@
+package com.vlad.kuzhyr.ratingservice.utility.constant;
+
+import lombok.AccessLevel;
+import lombok.NoArgsConstructor;
+
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public final class ExceptionMessageConstant {
+
+    public static final String RATING_NOT_FOUND_MESSAGE = "rating not found by id: %d";
+
+    public static final String RATINGS_NOT_FOUNDED_BY_PASSENGER_ID_MESSAGE = "ratings not founded by passenger id: %d";
```

Comment:
```suggestion
    public static final String RATINGS_NOT_FOUND_BY_PASSENGER_ID_MESSAGE = "ratings not found by passenger id: %d";
```

неправильный глагол) 
founded имеет немного другое значение

Reply:
заменил